Detections/W3CIISLog/ProxyShellPwn2Own.yml

```diff
@@ -0,0 +1,42 @@
+id: 968358d6-6af8-49bb-aaa4-187b3067fb95
+name: Exchange ProxyShell Pwn2Own
+description: |
+  'This query looks for suspicious request patterns to Exchange servers that fit patterns recently
```

Comment:
Any chance you can put a bit more in the description here.  For example, maybe the chain of high level events indicated in the blog?

Reply:
Improved description to include more detail